NEWS.md

```diff
@@ -95,7 +95,7 @@ Contributions from: Christian Kellner, David Rheinsberg
     format was added. Basic checks were added to the tests
     to check validation, loading and describing the new
     format.
-    The Manifest Marco Preprocessor (MPP) now also supports
+    The Manifest Macro Preprocessor (MPP) now also supports
```

Comment:
It is actually just called `Manifest Preprocessor`. No Marco involved. It is `MPP`, not `MMP`.

Reply:
Ah, I thought the MPP stood for "Macro Pre Processor" or that the macro was there for a reason.  Clearing up.